autopts/wid/mcp.py

```diff
@@ -341,24 +350,29 @@ def hdl_wid_20110(params: WIDParams):
     addr_type = btp.pts_addr_type_get()
     addr = btp.pts_addr_get()
 
-    if "0x00F3" in params.description:
+    """ PTS is providing handle value instead of expected action so we map it here based on
```

Comment:
Is the description of this WID correct? Seems more related to MICP than MCP; MCP isn't related to audio input control point

Reply:
I think this is copy'n'paste error in those descriptions